packages/components/src/tooltip/index.tsx

```diff
@@ -45,8 +47,24 @@ function Tooltip( props: TooltipProps ) {
 		}
 	}
 
+	// Compute tooltip's placement:
+	// - give priority to `placement` prop, if defined
+	// - otherwise, compute it from the legacy `position` prop (if defined)
+	// - finally, fallback to the DEFAULT_PLACEMENT.
```

Comment:
Now that we're inlining the default value, we can update this comment to say `bottom` instead of `DEFAULT_PLACEMENT`

Reply:
Good catch!